modules/components/src/AdvancedSqonBuilder/sqonPieces/FieldOp.js

```diff
@@ -56,10 +58,11 @@ export default props => {
                 </span>
                 <span className={`opName`}>{` is ${
                   (Array.isArray(value) && value.length > 1) ||
-                  RANGE_OPS.includes(op)
+                  ((RANGE_OPS.includes(op) || TERM_OPS.includes(op)) && !(isEqual(value,["true"]) || isEqual(value,["false"])))
```

Comment:
@jaouad-benassila This probably makes sense, but I don't understand? `!(isEqual(value,["true"]) || isEqual(value,["false"])`

Reply:
TERM_OPS is used for TermFilter
RANGE_OPS is used for RangeFilter
!(isEqual(value,["true"]) || isEqual(value,["false"]) is used to exclude boolean filter